woocommerce/woocommerce
# Add/36075 render fields sections from php config

### All Submissions:






### Changes proposed in this Pull Request:



Closes # .






### How to test the changes in this Pull Request:



1. In your `mu-plugin` add this action:
```php
/**
 * Add Example items to WooCommerce Navigation
 */
function add_product_form_field() {
    if (
        ! method_exists( '\Automattic\WooCommerce\Internal\Admin\ProductForm\FormFactory', 'add_field' )
    ) {
        return;
    }

    \Automattic\WooCommerce\Internal\Admin\ProductForm\FormFactory::add_field(
        'test_new_field',
        'woocommerce',
        array(
            'type'       => 'text',
            'section'    => 'general/details',
            'order'      => 2,
            'properties' => array(
                'name'       => 'name',
                'label'      => 'label',
            )
        )
    );

    \Automattic\WooCommerce\Internal\Admin\ProductForm\FormFactory::add_field(
        'test_new_section_field',
        'woocommerce',
        array(
            'type'       => 'text',
            'section'    => 'test_new_section',
            'properties' => array(
                'name'       => 'new',
                'label'      => 'New field',
            )
        )
    );

    \Automattic\WooCommerce\Internal\Admin\ProductForm\FormFactory::add_section(
        'test_new_section',
        'woocommerce',
        array(
            'type'       => 'text',
            'title'      => 'New title',
            'location'   => 'tab/general',
            'order'      => 4,
        )
    );
}
add_action( 'init', 'add_product_form_field' );
```
2. Enable the `new-product-management-experience` feature flag.
3. Go to **WooCommerce > Home** and open your console.
4. And check if your newly added field and section correctly show up in the correct order.




### Other information:




### FOR PR REVIEWER ONLY:
